### DISCUSSION TITLE
Some characters and icons not displaying

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#19017] [Q&A]
### Type

Cosmetic

### Bug description

Some of the icons and characters are not drawn when running the game. This doesn't appear when i launch the game for the first time after booting, only on following launches.

I am using laptop with Fedora 36, CPU Intel i5-8300H (8) @ 4.000GHz and discrete NVIDIA GeForce GTX 1050 Ti Mobile GPU with proprietary Nvidia drivers. I wasn't able to reproduce the issue using onboard Intel GPU.

### Screenshots or videos

![Snímek obrazovky z 2022-07-06 10-28-48](https://user-images.githubusercontent.com/15785828/177508462-468ed5ec-ac4f-4474-afda-b97e912f0169.png)
![Snímek obrazovky z 2022-07-06 10-34-28](https://user-images.githubusercontent.com/15785828/177508500-4f1f0e4f-8cb1-46a5-9327-3643bdd1b09d.png)
![Snímek obrazovky z 2022-07-06 10-34-40](https://user-images.githubusercontent.com/15785828/177508501-c1e3b849-7083-42eb-8871-a71e443b8e74.png)

### Version

2022.704.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/9053142/runtime.log)

**Comment by peppy** [comment]
This sounds like a GPU driver bug. We've seen this happen before but have never really found a resolution. You may be able to find some resolve by either using your other GPU (as you mentioned works), using windows, switching your drivers for another version, or waiting until we add vulkan support for another graphics pathway.
